src/app/components/result-list/result-list.component.html

```diff
@@ -1,7 +1,16 @@
-<div class="container">
+<div class="container" *ngIf="!characters; else results">
   <section class="results">
-    <ng-container *ngFor="let character of characters">
-      <app-character-result-box [character]="character"></app-character-result-box>
-    </ng-container>
+    <h1 class="no-results">
+      No se encontraron resultados asociados a la búsqueda
+    </h1>
   </section>
 </div>
+
+<ng-template #results>
+  <div class="container" *ngIf="characters!.length > 0">
```

Comment:
Why this validation?

Reply:
Because, I don't want to show the element if the characters array is empty. []
However, I want to show the no results message only if the characters array is null.
So:
1. [... ,... ,..,] ----> show results.
2. [] ---> don't show anything.
3. null ---> Show no results.

I'm not sure if this the best approach though.

Reply:
Great, I understand